content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -17,11 +17,15 @@ further_reading:
       text: 'Correlate request logs with traces automatically'
 ---
 
-## Automatic Trace ID injection
+## Automatic log injection
 
-Enable injection in the .NET Tracer’s [configuration][1] by setting `DD_LOGS_INJECTION=true` through environment variables or the configuration files.
+Enable injection in the .NET tracer’s [configuration][1] by setting `DD_LOGS_INJECTION=true` through environment variables or the configuration files.
```

Comment:
Was this change intentional? In dotnet-land we capitalize ".NET Tracer" and "Agent" as product names. I'm just looking for some consistency because it's capitalized again in the next line below.

Reply:
Good catch, this was probably done by mistake. I'll run through the tracing docs to make sure we're keeping it consistent.